### DISCUSSION TITLE
does E::Time have unit of nanoseconds?

### DISCUSSION THREAD

**Discussion by livelykitten** [ANLAB-KAIST/KENSv3#158] [Project Q&A 2. Transport Layer (TCP)] [answered]
I guess it is in nanoseconds, looking at https://github.com/ANLAB-KAIST/KENSv3/wiki/Tip:-Using-a-Timer
but I am not confident with my guess..

Thank you

**Comment by Pusnow** [comment, marked as answer]
Yes, it is nanoseconds. Thank you for the comment.

**Reply by livelykitten** [reply]
Thank you for your answer!
